doc/source/methods/utility_functions.rst

```diff
@@ -0,0 +1,15 @@
+.. _ref_utility_functions:
+
+Utility functions
+==================
+
+.. currentmodule:: ansys.motorcad.core.rpc_client_core
+
+
+.. autosummary::
+   :toctree: _autosummary_utility_functions
+
+   MotorCADError
+   set_default_instance
```

Comment:
Can we improve the docstrings for these to match the rest of our functions. e.g. params/return documented